### PR TITLE
BCMP / OpenShift - Fingerprint Letter Job Status

### PR DESCRIPTION
**Pull request by peggy-quartech**
# Description

This PR includes the following proposed change(s):

- spdbt-2465(BCMP / OpenShift - Fingerprint Letter Job Status)

### CODE REVIEW

File: src/Spd.Presentation.Dynamics/Controllers/PrintingController.cs [hunk 1]
```diff
@@ -43,19 +43,16 @@ public class PrintingController(IMediator mediator, IMapper mapper) : SpdControl
...
+    /// update the event queue..
     /// </summary>
-    /// <param name="jobId"></param>
+    /// <param name="eventId"></param>
     /// <param name="ct"></param>
     /// <returns></returns>
-    [HttpGet("api/printjobs/{jobId}/status")]
-    public async Task<PrintJobStatusResponse> GetPrintJobStatus(string jobId, CancellationToken ct)
+    [HttpGet("api/printjobs/{eventId}/status")]
+    public async Task<Results<Ok<string>, BadRequest>> GetPrintJobStatus(Guid eventId, CancellationToken ct)
```
Comment (ruaraya) [inline src/Spd.Presentation.Dynamics/Controllers/PrintingController.cs:53]: How do we know the status if we only get the id?

**Comment by ruaraya** [comment]
I think that the ticket you put in the description is the wrong one

File: src/Spd.Presentation.Dynamics/Controllers/PrintingController.cs (hunk 1 above)
Comment (peggy-quartech) [reply src/Spd.Presentation.Dynamics/Controllers/PrintingController.cs:53]: that's a good question. Actually, we update the status in the spd_eventqueue table.
But I think I should change here is return the status.
Let me fix that. thx.

File: src/Spd.Manager.Printing/PrintingManager.cs [hunk 2]
```diff
@@ -41,29 +54,54 @@ public async Task<string> Handle(StartPrintJobCommand request, CancellationToken
...
+        }
+        catch (Exception ex)
+        {
+            ResultResponse result = new()
+            {
+                PrintJobId = eventResp.JobId,
+                Status = JobStatusCode.Fail,
+                Error = $"{ex.Message} {ex.InnerException?.Message}"
+            };
+            //await UpdateResultInEvent(result, request.EventId, cancellationToken); //we should not update job status if we read status failed.
```
Comment (ruaraya) [inline src/Spd.Manager.Printing/PrintingManager.cs:102]: Do we need this commented out code?

Comment (peggy-quartech) [reply src/Spd.Manager.Printing/PrintingManager.cs:102]: leave it there. 
It is possible that Mano want it.

**Review by ruaraya — APPROVED** [review]

**Pull request merged**